src/views/Main/Header/Header.js

```diff
@@ -13,23 +13,24 @@ import NavBarItemDesktop from './NavBarItemDesktop';
 import NavBarItemTablet from './NavBarItemTablet';
 import LoginButton from './LoginButton';
 import PartnerButton from './PartnerButton';
-import logo from '../../../images/terravisu-logo.svg';
 
 import infoSign from '../../../images/info-sign.svg';
 import logIn from '../../../images/log-in.svg';
 import logOut from '../../../images/log-out.svg';
+import defaultLogo from '../../../images/terravisu-logo.svg';
 
 import './styles.scss';
 
-const generateMenus = (authenticated, views) => {
+const generateMenus = (authenticated, views, settings = {}) => {
+  const hasCustomLogo  = settings.theme && settings.theme.logo && settings.theme.logo.length;
```

Comment:
Do you need the last part about the length ?

Reply:
In case the sting is empty, it returns the default logo